### PR TITLE
documentation link fixes

### PR DESCRIPTION
**Pull request by rj-taylor**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Please read [the contributing documentation](https://docs.librenms.org/#Developing/
+Please read [the contributing documentation](https://docs.librenms.org/Support/Cleanup-options/
```
Comment (PipoCanaja) [inline CONTRIBUTING.md]: looks like a copy paste error here

Comment (rj-taylor) [reply CONTRIBUTING.md]: That's embarrassing!  I've fixed it in my master.  Do I submit a new pull-request?  (First day on GitHub.)

Comment (rj-taylor) [reply CONTRIBUTING.md]: I think I figured it out.  "Re-request review."

**Comment by PipoCanaja** [comment]
Perfect. Thanx @rj-taylor !

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-58-release-changelog-november-2019/10224/1

**Pull request merged**